raspberry/bzzz/sensors/gnss.py

```diff
@@ -187,34 +344,73 @@ def latitude(self):
                                                  cursor=self.__cursor)
 
     @property
-    def longitude(self):
+    def relative_east(self):
         """
         Returns Longitude position in decimal
         """
         with self.__lock:
             return self.__data_processor.process(self.__values_cache[:, 1], 
                                                  cursor=self.__cursor)
+        
+    @property
+    def relative_down(self):
+        """
+        Returns Longitude position in decimal
+        """
+        with self.__lock:
+            return self.__data_processor.process(self.__values_cache[:, 2], 
+                                                 cursor=self.__cursor)
 
     @property
     def altitude(self):
         """
-        Returns the Altitude position
+        Returns the Altitude of the quadcopter in meters based of the relative 
```

Comment:
What is the difference between `altitude` and `relative_down`? They seem to be almost identical.

Reply:
What do you mean by "direction changed"? In any case, shouldn't there be a single "relative down" measurement from the GNSS? Also, since the caller has provided a filter, they want it to be applied to all measurements, otherwise they can use a `NoFilter`.

Reply:
For `relative_down` the value decreases the higher the quadcopter flies. (z axis pointing down)
for `altitude` the value increases the higher the quadcopter flies like the other sensors (z axis pointing up)

Reply:
Is this useful?